modules/virt-linux-bridge-nad-port-isolation.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-linux-bridge-nad-port-isolation.adoc_{context}"]
+= Enabling port isolation for a Linux bridge NAD
+
+You can enable port isolation for a Linux bridge network attachment definition (NAD) so that virtual machines (VMs) or pods that run on the same virtual LAN (VLAN) can operate in isolation from one another. The Linux bridge NAD creates a virtual bridge, or _virtual switch_, between network interfaces and the physical network. You can then configure port numbers on the virtual bridge so that only specific traffic can reach each port. 
```

Comment:
@dfitzmau This is not correct statement, you can remove `You can then configure port numbers on the virtual bridge so that only specific traffic can reach each port.`

@yossisegev Could you help review some settings specific to VM workloads?